common/signal.edl

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+/*
+**==============================================================================
+**
+** signal.edl:
+**
+**     This file declares OCALLs and ECALLs needed by the enclave to implement
+**     functions in POSIX signal.h
```

Comment:
nit: missing period.

Reply:
@mingweishih for trivial changes, it's worth using suggestion blocks: 
![image](https://user-images.githubusercontent.com/4016369/76838018-72600800-682b-11ea-814d-f25c5159fbc9.png)

The PR submitter can just "approve" and the change goes in straight away, it's often just as quick if not quicker than writing the comment :)